### DISCUSSION TITLE
Collision with ground

### DISCUSSION THREAD

**Discussion by MariaALJ** [JSBSim-Team/jsbsim#869] [Q&A]
Hello, 

I installed the JSBSim plugin on a personal project where my aim is to control the flight path. It has been working pretty well in flight mode but i have a problem when it comes to the take-off : I put a simple plane to simulate the ground in order to control take-off. The aircraft starts at initial coordinates (0,0,0) but when I hit play the aircraft sinks into the ground to the coordinate z=-335 and whatever initial z coordinate I put, it will always go back to -335 and then the aircraft can't achieve the take-off.

https://user-images.githubusercontent.com/116007762/228212831-e6dbcd8f-3925-40d2-ae60-a3729ba3184e.mp4

![Capture](https://user-images.githubusercontent.com/116007762/228213445-01cbcd41-ec1e-48b8-9c1f-3d24515cdc19.PNG)

Any ideas of what could be the cause of this to happen?

Thank you in advance.

**Comment by gallonmate** [comment]
It's a bug that was introduced in UE5.1
There is already an Issue started here: #786 
It's not fixed yet but for now you can work around the issue by doing this:

The work around for that is to set Start on Ground to false in BP Airliner in the Details of the level editor.
![image](https://user-images.githubusercontent.com/28702753/210048089-86e37015-5178-4d5a-b1ae-58af62fff9d4.png)
For some smaller aircraft, seems you can still use Start on Ground, as it's not broken for all aircraft.
